third-culture-software/bhima
# Add default patient discharge types

Add the default patient `discharge_type`s to all sites.

**Suggestions for Testing**

- Use the IMCK database (which has the `discharge_type` table but not  does not have data)
- Edit .env appropriately
- Download the latest IMCK database
- Run the migration script
- Use MySQL to verify the additions
   - mysql -u bhima -p --database=imck
   - select * from discharge_type  ==> Should show 9 entries
- Run bhima (yarn dev)
- Go to the Hospital > Patient Registry
- In the patient registry, select a period so you see patients
- Click on a patient NAME (clicking on the Reference gives the patient ID document, which does not allow the next step)
- In the Patient Details, click on the [Admit Patient] button (bottom right)
   - Fill out the form and click [Submit]
   - The button on the bottom right of the Patient Details should change to [Discharge Patient]
- Click on the [Discharge Patient] button
   - You should see the form with 9 options for discharge type.  Click on, add diagnosis and [Submit]
   - The submission should be succesful
- Visit the Hospital > Visits Registry
   - You should see the  visit created above listed with the discharge type
